### PR TITLE
[MDS-5944 ] application submission success fail page

### PR DESCRIPTION
**Pull request by asinn134**
## Objective 

[MDS-5944 ](https://bcmines.atlassian.net/browse/MDS-5944 )

-This contains the same changes found in the previous pr (https://github.com/bcgov/mds/pull/3171)
-There are a few other changes, but the biggest ones are around the `facility_coords_source` property. It was found that when we make a submission to the AMS API, we would keep getting a 400 status code error with the message `Business Errors Exist: Queued errors exist`. 

The cause turned out to be because in the Major project's Location Access and Land Use section, the Source of Data drop down, when say for example Survey is selected, we are actually sending the value "SUR" instead of "Survey" and AMS API only accepts the following values for the `facility_coords_source` property:

Screenshot (374)

### CODE REVIEW

File: services/core-web/src/styles/components/Callout.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.bcgov-callout--success>p,
+.bcgov-callout--warning>p,
+.bcgov-callout--danger>p {
+    margin-bottom: 0;
+}
+
+.bcgov-callout h1,
+h2,
```
Comment (taraepp) [inline services/core-web/src/styles/components/Callout.scss]: I see you're bringing this file over to CORE!

An issue that I noticed with this on Minespace: this selector doesn't work as intended. They meant for every `h1`, `h2`, etc, _within_ `.bcgov-callout` to be selected, but what actually happened was they selected the `h1` within bcgov-callout and also _every single other_ `h2`-`h4`! I changed the selector, and then made some other changes that sonarcloud or eslint wanted, in the PR that I put up today.

File: services/core-web/src/components/mine/Projects/ProjectSummary.tsx [hunk 2]
```diff
@@ -130,20 +137,42 @@ export const ProjectSummary: FC = () => {
...
+    );
+
+    await handleFetchData();
+    if (
+      tab === PROJECT_SUMMARY_WITH_AMS_SUBMISSION_SECTION &&
+      amsFeatureEnabled &&
+      projectSummaryResponse
+    ) {
+      const { data } = projectSummaryResponse;
+      const authorizations = data?.authorizations;
```
Comment (taraepp) [inline services/core-web/src/components/mine/Projects/ProjectSummary.tsx]: should probably be `data?.authorizations ?? []` for the array operations below

**Review by taraepp — COMMENTED** [review]
Looks good overall. 👍 
Couple minor things.

File: services/core-web/src/styles/components/Callout.scss (hunk 1 above)
Comment (asinn134) [reply services/core-web/src/styles/components/Callout.scss]: Thanks for the heads up. I adjusted this to be aligned with the change you made in your pr.

File: services/core-web/src/components/mine/Projects/ProjectSummary.tsx (hunk 2 above)
Comment (asinn134) [reply services/core-web/src/components/mine/Projects/ProjectSummary.tsx]: Thanks, this has been adjusted now

**Review by taraepp — APPROVED** [review]

**Review by simensma-fresh — APPROVED** [review]

**Pull request merged**